modules/react/layout/spec/mergeStyles.spec.tsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
 
-import {cache} from '@emotion/css';
-import {jsx, CacheProvider} from '@emotion/react';
+import {jsx} from '@emotion/react';
 import styled from '@emotion/styled';
 import {render as rtlRender, screen} from '@testing-library/react';
 
-// We need to force Emotion's cache wrapper to use the cache from `@emotion/css` for tests to pass
-const CacheWrapper = props => <CacheProvider value={cache} {...props} />;
-// @ts-ignore We want the types to be the same, but I don't care to fix the error
-const render: typeof rtlRender = (ui, options) =>
-  rtlRender(ui, {wrapper: CacheWrapper, ...options});
-
 import {mergeStyles} from '@workday/canvas-kit-react/layout';
 import {createStyles} from '@workday/canvas-kit-styling';
+import {CanvasProvider} from '@workday/canvas-kit-react/common';
+
+// We need to force Emotion's cache wrapper to use the cache from `@workday/canvas-kit-styling`
+// @ts-ignore We want the types to be the same, but I don't care to fix the error
+const render: typeof rtlRender = (ui, options) =>
+  rtlRender(ui, {wrapper: CanvasProvider, ...options});
```

Comment:
The `CanvasProvider` handles setting the correct cache object, so we should use `CanvasProvider` instead a custom `CacheProvider`